src/python/qeqiskit/backend/backend.py

```diff
@@ -43,6 +44,8 @@ def __init__(
         Returns:
             qeqiskit.backend.QiskitBackend
         """
+        self.number_of_circuits_run = 0
+        self.number_of_jobs_run = 0
```

Comment:
The preferred behavior would be `super().__init__(n_samples=n_samples)` and then we can remove line `50` as well

Reply:
Addressed in [d81040](https://github.com/zapatacomputing/qe-qiskit/pull/18/commits/d8104090971fb60f46de371cafe4e6e745c211d3).